install/froxlor.sql.php

```diff
@@ -613,7 +613,9 @@
 	('system', 'mdalog', '/var/log/mail.log'),
 	('system', 'mtalog', '/var/log/mail.log'),
 	('system', 'mdaserver', 'dovecot'),
+	('system', 'mda_reload_command', 'service dovecot restart'),
```

Comment:
missing `mda_conf_dir` and `mta_conf_dir`, also incrementing `db_version` is required for correct update handling